### PR TITLE
fix: golangci.yml to keep quality of codes

### PR DESCRIPTION
**Pull request by mnagaa**
# Summary 

golangci lint does not work because of including unnecessary indent in the `.golangci.yml`.
Fixing the broken `.golangci.yml` and invalid expressions in the existing go files are done in this PR.

Notice that linters commented out in the file yaml file need fix many files, but they should work in the future.

### CODE REVIEW

**Comment by mnagaa** [comment]
@sfc-gh-swinkler Could you please review this change?

File: pkg/resources/tag_association.go [hunk 1]
```diff
@@ -116,9 +119,26 @@ func CreateTagAssociation(d *schema.ResourceData, meta interface{}) error {
 		return errors.Wrapf(err, "error associating tag to object: [%v] with command: [%v], tag_id [%v]", objectIdentifier, q, tagID)
 	}
 
-	_, err = snowflake.ListTagAssociations(builder, db)
-	if err != nil {
-		return errors.Wrap(err, "error validating tag association")
+	skipValidate := d.Get("skip_validation").(bool)
```
Comment (sfc-gh-swinkler) [inline pkg/resources/tag_association.go:122]: im wondering where this came from. could you please rebase to main?

**Comment by sfc-gh-swinkler** [comment]
could you please rebase to main? theres some changes to the tag_association resource i am thinking probably came from an old PR

File: pkg/resources/tag_association.go (hunk 1 above)
Comment (mnagaa) [reply pkg/resources/tag_association.go:122]: Sure! I think this came from other PR and I failed to operate merging other one. I'll check this carefully then fix around this.

Comment (mnagaa) [reply pkg/resources/tag_association.go:122]: @sfc-gh-swinkler Again, thank you for the review. I checked changes around these lines and found just my deleting blank two lines. Does this problem come from some where else...?

**Comment by mnagaa** [comment]
@sfc-gh-swinkler Fixing docs by updating docs, could you please rerun workflow?

**Review by sfc-gh-swinkler — APPROVED** [review]

**Comment by sfc-gh-swinkler** [comment]
@mnagaa thank you for your contribution1

**Pull request merged**